docs/docs/deploy/prerequisites.md

```diff
@@ -32,6 +32,16 @@ configuration file at `/etc/firezone/firezone.rb`. See here for a guide on how
 to do so:
 [https://eff-certbot.readthedocs.io/en/stable/using.html#manual](https://eff-certbot.readthedocs.io/en/stable/using.html#manual)
 
+## Security Group and Firewall Settings
+
+By default, Firezone requires ports `80`, `443`, and `51820` to be open.
```

Comment:
Clarification:

By default, Firezone requires ports `80/tcp`, `443/tcp`, and `51820/udp` to be open. This can change based on what you've configured in the configuration file. See the
 [configuration file reference]({% link docs/reference/configuration-file.md %})
 for details.